### PR TITLE
[Symfony UX Map] Create recipes for the package and its bridges

### PR DESCRIPTION
**Pull request by Kocal**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | symfony/symfony-docs#...

Adding recipies for the new Symfony UX Map package and its Bridges, currently being reviewed in https://github.com/symfony/ux/pull/1937.
I've specified version 2.19, but it may be 2.20 instead, let's keep it in draft then.

### CODE REVIEW

**Comment by AlDaFlux** [comment]
I try it on a php8.3 / Synfony 6.4

```
ubuntu@vps-95f47936:~/html/sandbox/dev$ echo $SYMFONY_ENDPOINT
https://raw.githubusercontent.com/symfony/recipes/flex/pull-1329/index.json
ubuntu@vps-95f47936:~/html/sandbox/dev$ composer req 'symfony/flex:^1.16'
./composer.json has been updated
Running composer update symfony/flex
Loading composer repositories with package information
Updating dependencies
Nothing to modify in lock file
....
ubuntu@vps-95f47936:~/html/sandbox/dev$ composer req 'symfony/ux-map:^2.19' 'symfony/ux-map-google:^2.19' 'symfony/ux-map-leaflet:^2.19'
./composer.json has been updated
Running composer update symfony/ux-map symfony/ux-map-google symfony/ux-map-leaflet
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires symfony/ux-map, it could not be found in any version, there may be a typo in the package name.
  Problem 2
    - Root composer.json requires symfony/ux-map-google, it could not be found in any version, there may be a typo in the package name.
  Problem 3
    - Root composer.json requires symfony/ux-map-leaflet, it could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
ubuntu@vps-95f47936:~/html/sandbox/dev$

```

**Review by maxhelias — APPROVED** [review]

**Comment by fabpot** [comment]
@Kocal  Not a draft anymore, right?

**Comment by Kocal** [comment]
@fabpot it should be alright now

**Review by fabpot — DISMISSED** [review]

**Comment by fabpot** [comment]
We now need a 2.19 version of UX. Can I make the releases?

**Comment by fabpot** [comment]
@kbond maybe?

**Comment by kbond** [comment]
@fabpot, can we do it Friday?

**Comment by Kocal** [comment]
@fabpot @kbond I've rebased the PR, the recipes are ready to be merged if you want to :)

Checks fails because it's PHP 8.3 only tho

File: symfony/ux-map/2.19/post-install.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+  * You're near ready to use the Symfony UX Map component. You can create your first map
+    by using the <info>Map</info> class, but you will not be able to render it yet until you:
+    
+    1. Install the Symfony UX Map Bridge you want to use, they can be
+       found at <comment>https://symfony.com/bundles/ux-map/current/index.html#available-renderers</>.
+    2. Uncomment the <info># renderer: '%env(UX_MAP_DSN)%'</> line
```
Comment (kbond) [inline symfony/ux-map/2.19/post-install.txt:6]: Is this outdated? The line is not commented.

**Review by fabpot — APPROVED** [review]

**Pull request merged**